technique/muon/tektronix_osc_screenshots.py

```diff
@@ -0,0 +1,91 @@
+import os
+import requests
+import sys
+import shutil
+
+from datetime import datetime
+from io import BytesIO
+from PIL import Image
+from time import sleep
+from zipfile import ZipFile
+
+sys.path.insert(0, os.path.abspath(os.path.join(r"C:\\", "Instrument", "scripts")))
+sys.path.insert(0, os.path.abspath(os.path.join(os.environ["KIT_ROOT"], "ISIS", "inst_servers", "master")))
+
+from genie_python import genie as g
+from genie_python.channel_access_exceptions import UnableToConnectToPVException
+from server_common.helpers import register_ioc_start
+
+register_ioc_start("BGRSCRPT_03")
+
+g.set_instrument(None)
+
+# User editable constants
+DEVICE_IP = "130.246.50.7"
```

Comment:
Unless this is an expected default probably better to have this be None or an empty string so you can check that its been preoperly set.